File: src/SourceCodes/CourseSchedule.java
package SourceCodes;

import java.util.ArrayList;

public class CourseSchedule {

    private Day courseDay;
    private ArrayList<Hour> courseHours;

    public CourseSchedule(Day courseDays,ArrayList<Hour> courseHours) {
        this.courseDay = courseDays;
        this.courseHours = courseHours;
    }

    public Day getCourseDay() {
        return courseDay;
    }

    public void setCourseDay(Day courseDay) {
        this.courseDay = courseDay;
    }

    public ArrayList<Hour> getCourseHours() {
        return courseHours;
    }

    public void setCourseHours(ArrayList<Hour> courseHours) {
        this.courseHours = courseHours;
    }

    
}

File: python_development/CourseSchedule.py
class CourseSchedule:
    def __init__(self, course_day, course_hours):
        self.course_day = course_day
        self.course_hours = course_hours

    def getCourseDay(self):
        return self.course_day

    def setCourseDay(self, course_day):
        self.course_day = course_day

    def getCourseHours(self):
        return self.course_hours

    def setCourseHours(self, course_hours):
        self.course_hours = course_hours